README.md

```diff
@@ -81,8 +81,8 @@ UDP Port used to send commands (start/stop) and receive the corresponding replie
 _sensor_ip_ (_string_, default: "192.168.0.10")<br/>
 IP-Address of safety laser scanner.
 
-_frame_id_ (_string_, default: "scanner")<br/>
-Identifier used for transformations within ROS environment.
+_prefix_ (_string_, default: "laser_1")<br/>
```

Comment:
is this a prefix also used by other vendors or could we name it like psenscan_1 to have it related to the pilz product?

Reply:
other vendors use a similar method. I think we have the psenscan branding already in many places.